src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -74,7 +74,9 @@ export class Conversations extends React.Component<Props, State> {
     }
     const conversations = me.conversations.edges
       .filter(({ node }) => {
-        return node.last_message
+        if (node) {
+          return node.last_message
```

Comment:
this can be `node && node.last_message` - this would have got caught by strictNullChecking in the tsconfig

Reply:
as in, `return node && node.last_message`? 

how does that not return `node` instead of `node.last_message` ._. #jswizardry

Reply:
```
~/dev/projects/artsy/js/sites/force update_blog_url
❯ node
> 1 && 2
2
```